test/test_torch.py

```diff
@@ -201,7 +201,7 @@ def test_horovod_allreduce_multi_gpu(self):
 
         iter = 0
         dtypes = [torch.cuda.IntTensor, torch.cuda.LongTensor,
-                  torch.cuda.FloatTensor, torch.cuda.DoubleTensor]
+                  torch.cuda.HalfTensor, torch.cuda.FloatTensor, torch.cuda.DoubleTensor]
```

Comment:
Let's add `torch.cuda.HalfTensor` to other tests as well?  It implies GPU by itself.